jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -214,6 +218,7 @@ async def _async_start_kernel(self, kernel_id=None, path=None, **kwargs):
             if path is not None:
                 kwargs["cwd"] = self.cwd_for_path(path, env=kwargs.get("env", {}))
             if kernel_id is not None:
+                assert kernel_id is not None
```

Comment:
I'm curious under what circumstances this assertion will fire.

Reply:
None, this is just for mypy/type checking. As I aded types to kwargs above, mypy complains that in the line after kernel_id can be None. Which is obviously not true.